packages/rum/src/domain/record/observers/mutationObserver.ts

```diff
@@ -49,6 +49,14 @@ export type RumMutationRecord =
   | RumAttributesMutationRecord
   | RumChildListMutationRecord
 
+export function initMutationObserver(
+  cb: MutationCallBack,
+  configuration: RumConfiguration,
+  shadowRootsController: ShadowRootsController
+) {
+  return startMutationObserver(cb, configuration, shadowRootsController, document)
+}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about exposing a single method and calling it with `document` from `initObservers`?